src/events.ts

```diff
@@ -1,28 +1,38 @@
 import { emit, EventCallback, listen } from '@tauri-apps/api/event';
 
-import { PaneEditorId } from './atoms/types/PaneGroup';
+import { PaneEditorId, PaneId } from './atoms/types/PaneGroup';
 import { NotificationItemType } from './notifications/types';
 
 interface RefStudioEvents {
-  'refstudio://menu/file/close': undefined;
-  'refstudio://menu/file/save': undefined;
+  // Menu actions (Note that these should have undefined payload)
   'refstudio://menu/file/new': undefined;
+  'refstudio://menu/file/save': undefined;
+  'refstudio://menu/file/close': undefined;
+  'refstudio://menu/file/close/all': undefined;
   'refstudio://menu/settings': undefined;
   'refstudio://menu/references/open': undefined;
   'refstudio://menu/references/upload': undefined;
   'refstudio://menu/references/export': undefined;
   'refstudio://menu/view/notifications': undefined;
+  'refstudio://menu/debug/console/clear': undefined;
+  // Editor actions
   'refstudio://editors/close': PaneEditorId;
+  'refstudio://editors/move': { editor: PaneEditorId; toPaneId: PaneId };
```

Comment:
Maybe you could rename it `fromPaneEditorId` to be more explicit? Or at least `paneEditorId`, in order not to get confused with the type of the attribute